web/src/routes/TimelineView.svelte

```diff
@@ -5,18 +5,21 @@
 	import { author, isMuteEvent } from '../stores/Author';
 	import Loading from './Loading.svelte';
 	import EventComponent from './timeline/EventComponent.svelte';
+	import { nip19 } from 'nostr-tools';
```

Comment:
すみません import が抜けてました。
```suggestion
	import { nip19 } from 'nostr-tools';
	import { goto } from '$app/navigation';
```